docs/custom-contract/README.md

```diff
@@ -0,0 +1,57 @@
+# Custom contract implementation
+See `@prescott/contract` for a list of available contracts.
+
+## Types
+First of all, the process depends on the source of the contract. Now there are two of such: `'npm' | 'file'`.
```

Comment:
it would be nice to have git option

Reply:
I thought about the `file` option but decided to postpone it, as some safety mechanism is required here
(I didn't want to dive into `vm`)